spec/dishes_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'dishes'
+
+describe Dishes do
+  it "has a list of a dishes" do
+    expect(subject.dish_list).to eq :FishAndChips => 5, :MeatAndTwoVeg => 7
```

Comment:
You can use the syntactical sugar for hashes here: "FishAndChips: 5, MeatAndTwoVeg: 7"